include/invert_quda.h

```diff
@@ -673,6 +733,70 @@ namespace quda {
     virtual bool hermitian() { return true; } /** CG is only for Hermitian systems */
   };
 
+  /**
+     @brief  Generic solver with an accelerator.
```

Comment:
More documentation needed here.  My naive 🧠 would assume an accelerator is a GPU 😛

Reply:
https://github.com/lattice/quda/pull/1061/commits/96926239a741adb4fa5c8bab17dc6714e094af3b